### PR TITLE
Add get tags info method for releases

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

**Comment by lunny** [comment]
This will resolve go-gitea/gitea#490

File: repo_tag.go [hunk 1]
```diff
@@ -94,6 +95,85 @@ func (repo *Repository) GetTag(name string) (*Tag, error) {
...
+// 2016-10-14 20:54:25 +0200  (tag: translation/20161014.01) d3b76dcf2 Dirk Baeumer dirkb@microsoft.com Merge in translations
+func parseTag(line string, opt TagOption) (*Tag, error) {
+	line = strings.TrimSpace(line)
+	if len(line) < 40 {
+		return nil, nil
+	}
+
+	var err error
+	var tag Tag
+	var sig Signature
```
Comment (andreynering) [inline repo_tag.go]:
```go
var (
    err error
    tag Tag
    sig Signature
)
```

**Comment by andreynering** [comment]
Trusted LGTM

File: repo_tag.go (hunk 1 above)
Comment (lunny) [inline repo_tag.go]: Done.

**Comment by appleboy** [comment]
LGTM

**Pull request merged**